src/snovault/elasticsearch/create_mapping.py

```diff
@@ -133,10 +134,11 @@ def schema_mapping(name, schema):
             'type': field_type
         }
 
-        # these fields are unintentially partially matching some small search
-        # keywords because fields are analyzed by nGram analyzer
-        if name in NON_SUBSTRING_FIELDS:
-            sub_mapping['include_in_all'] = False
+        if name not in NON_SUBSTRING_FIELDS + TEXT_FIELDS:
```

Comment:
I think we do want to copy the `TEXT_FIELDS` to `_all`. The search code just looks in `_all` field now and expects it to be analyzed. We definitely want description etc. searchable.

Reply:
This is to make the search results as comparable as possible. Without the change, and without biosample term name filtering, the Elasticsearch 7 is going to show this experiment for "brain", "blood", or "breast" keywords  due to both words happening in the award description: https://www.encodeproject.org/experiments/ENCSR003IGU/?format=json. The result counts will be off.

Reply:
`The result counts will be off.` Are you sure? I know description is currently included in `_all` so it's free text searchable. 

For example putting `RNA Evaluation human tissue W72 left colon microRNA-seq from Mortazavi` into search box returns the experiment with that description: https://www.encodeproject.org/search/?searchTerm=RNA+Evaluation+human+tissue+W72+left+colon+microRNA-seq+from+Mortazavi

How will we maintain this functionality with ES7?

Reply:
You are right, the long query doesn't return if `TEXT_FIELD` is removed from `_all`. I will bring it back and we can compare queries.

Reply:
here is a demo for comparison: https://elasticsearch7.demo.encodedcc.org/

Reply:
This demo when indexed should be a closer match with the old one: https://elasticsearch7-similar-results.demo.encodedcc.org/